tsrc/test/cli/test_sync.py

```diff
@@ -108,6 +108,18 @@ def test_sync_not_on_master(
     assert message_recorder.find("not on the correct branch")
 
 
+def test_sync_with_force(
+    tsrc_cli: CLI, git_server: GitServer, workspace_path: Path
+) -> None:
+    git_server.add_repo("foo")
+    git_server.push_file("foo", "latest.txt", contents="1")
+    git_server.tag("foo", "latest")
+    tsrc_cli.run("init", git_server.manifest_url)
+    git_server.push_file("foo", "latest.txt", contents="2")
+    git_server.tag("foo", "latest", force=True)
+    tsrc_cli.run("sync", "--force")
+
```

Comment:
where is the assert?

Reply:
There's no need, we implicitly assert that the command does not fail

Reply:
We could patch the `tsrc_cli` feature so that calling `run` never throws and assert that it succeeds everywhere, but I'm not sure it's an improvement:

```python
def test_that_needs_a_workspace():
     # setup git server
     # create workspace
     assert tsrc_cli.run("init")  # <- this is not really an assertion

     # do the test
     assert tsrc_cli.run("someting")
```